eZ/Publish/API/Repository/Tests/UserServiceTest.php

```diff
@@ -2873,7 +2873,31 @@ public function testValidatePassword(string $password, array $expectedErrorr)
         $actualErrors = $userService->validatePassword($password, $context);
         /* END: Use Case */
 
-        $this->assertEquals($expectedErrorr, $actualErrors);
+        $this->assertEquals($expectedErrors, $actualErrors);
+    }
+
+    public function testValidatePasswordReturnsErrorWhenOldPasswordIsReused(): void
+    {
+        $password = 'P@blish123!';
+
+        $userService = $this->getRepository()->getUserService();
+        // Password expiration needs to be enabled
+        $contentType = $this->createUserContentTypeWithPasswordExpirationDate();
+
+        $user = $this->createTestUserWithPassword($password, $contentType);
+
+        /* BEGIN: Use Case */
+        $context = new PasswordValidationContext([
+            'contentType' => $contentType,
+            'user' => $user,
+        ]);
+
+        $actualErrors = $userService->validatePassword($password, $context);
+        /* END: Use Case */
+
+        $this->assertEquals([
+            new ValidationError('New password cannot be the same as old password', null, [], 'password'),
+        ], $actualErrors);
```

Comment:
```suggestion
        $this->assertEquals(
            [new ValidationError('New password cannot be the same as old password', null, [], 'password')], 
            $userService->validatePassword($password, $context)
        );
```
nitpick: without a temporary variable or with two of them